README.md

```diff
@@ -1 +1,1178 @@
-# ballerina-honeycomb
\ No newline at end of file
+  [![Build Status](https://travis-ci.org/ballerina-guides/ballerina-honeycomb)](https://travis-ci.org/ballerina-guides/ballerina-honeycomb)
+  
+## Integration with Honeycomb
+
+ Honeycomb is a tool used for investigating on how well your system product is working in various conditions (for example - high traffic). Through honeycomb we are able to collect data of your own software which can be broken down into various entities to observe its performance specifically. [Read more about Honeycomb.](#about-honeycomb)
+ 
+ 
+The following are the sections available in this guide.
+
+- [What you'll build](#what-you’ll-build)
+- [Prerequisites](#prerequisites)
+- [Implementation](#implementation)
+- [Testing](#testing)
+- [Configuration with Honeycomb](#testing-with-honeycomb)
+     - [Traces](#views-of-traces)
+     - [Metrics](#metrics)
+
+
+
+## What you’ll build 
```

Comment:
Can we have more explanation of the diagram and what you'll be implementing exactly here? There isn't enough detail here.

Reply:
Noted. I will add more explanation.